style.css

```diff
@@ -41,26 +41,40 @@ nav {
   display: none;
 }
 
-.hamburger_toggle {
+.hamburger-menu {
   display: flex;
   justify-content: flex-end;
   align-items: center;
   cursor: pointer;
   margin-left: 80%;
   margin-top: 0.75rem;
-  order: 0;
-  flex-grow: 0;
-  width: 2rem;
+  width: 2.5rem;
```

Comment:
- [ ] The hamburger menu is visible on desktop size. Please ensure it only displays on mobile. This will prevent display of 2 menus on the desktop size. 👍🏾 
![ham](https://github.com/mutonic/Portfolio_Microverse/assets/99656687/d284a346-3b2b-4035-a933-20f59102184b)